src/main/java/apaintus/util/ReflectionUtil.java

```diff
@@ -0,0 +1,38 @@
+package apaintus.util;
+
+import java.lang.reflect.Field;
+
+public class ReflectionUtil {
+	public static <V> V get(Object object, String fieldName) {
+		Class<?> objectClass = object.getClass();
+		while (objectClass != null) {
+			try {
+				Field field = objectClass.getDeclaredField(fieldName);
+				field.setAccessible(true);
+				return (V) field.get(object);
+			} catch (NoSuchFieldException e) {
+				objectClass = objectClass.getSuperclass();
+			} catch (Exception e) {
+				System.out.println("allo");
```

Comment:
i think doing this is bad, isn't it ;)  should be logged

Reply:
Quand on aura notre event logger

Reply:
au pire met un //TODO pour pas qu'on l'oublie

Reply:
Si tu parles du event logger de l'app 3, il ne sert qu'à logger les actions des utilisateurs, donc les erreurs ne sont pas log

Reply:
@boscoa S'il sert juste a ca, on va avoir log4j et notre event logger??? Tu veux pas un ErrorLogger dans ta chain of responsibility?

Reply:
C'est pas mentionné dans l'app 3 qu'il faut gérer les messages d'exception avec notre logger.

Reply:
Je pourrais ajouter en effet. Faudra juste que je le change en Singleton alors

Reply:
@boscoa Jvais laisser le spot vide pis on verra qu'est-ce que sonar en dit. Pas besoin de faire un error logger